### PR TITLE
ON-12177 # Added `encryptPdf` prop to hcms submission event

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: docs/forms.md [hunk 1]
```diff
@@ -752,33 +752,40 @@ return validatedForm
...
+| `isInfoPage`                                           | Yes                                               | `boolean`                                      | Whether or not the Form is an Info Page.                                                        |
+| `tags`                                                 | No                                                | `string[]`                                     | A list of tags used to categorise or describe the form.                                         |
+| `publishStartDate`                                     | No                                                | `string`                                       | The date and time (in ISO format) a form becomes available.                                     |
+| `publishEndDate`                                       | No                                                | `string`                                       | The date and time (in ISO format) a form becomes unavailable.                                   |
+
+contentTypeName: string,
+encryptedElementIds: ?Array<string>,
+encryptPdf?: boolean,
```
Comment (mymattcarroll) [inline docs/forms.md:788]: typo?

File: package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   },
   "devDependencies": {
     "@oneblink/release-cli": "^1.0.0",
-    "@oneblink/types": "github:oneblink/types",
+    "@oneblink/types": "github:oneblink/types#ON-12178",
```
Comment (mymattcarroll) [inline package.json:20]: This branch has been deleted.

**Comment by mymattcarroll** [comment]
Go to merge once those two things are fixed.

File: docs/forms.md (hunk 1 above)
Comment (Zaxist) [reply docs/forms.md:788]: Oops

**Pull request merged**